packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -85,15 +91,20 @@ const Trade = observer(() => {
                         <TradeParametersContainer>
                             <TradeParameters />
                         </TradeParametersContainer>
-                        <section className='trade__chart' style={{ height: dynamic_chart_height }} ref={chart_ref}>
-                            <TradeChart />
-                        </section>
+                        <div className='trade__chart-tooltip'>
+                            <section className='trade__chart' style={{ height: dynamic_chart_height }} ref={chart_ref}>
+                                <TradeChart />
+                            </section>
+                        </div>
                         {isAccumulatorContract(contract_type) && <AccumulatorStats />}
                     </div>
-                    <TradeParametersContainer is_minimized_visible={is_minimized_params_visible} is_minimized>
-                        <TradeParameters is_minimized />
-                    </TradeParametersContainer>
-                    <PurchaseButton />
+                    <div className='trade__parameter'>
+                        <TradeParametersContainer is_minimized_visible={is_minimized_params_visible} is_minimized>
+                            <TradeParameters is_minimized />
+                        </TradeParametersContainer>
+                        <PurchaseButton />
+                    </div>
+                    {!guide_dtrader_v2 && is_logged_in && <OnboardingGuide type='trade_page' />}
```

Comment:
are we meant to not show guide when logged out ?

Reply:
Yes, I think it doesn't make sense to show the guide to non-logged in users. Plus, there is also Guide for Positions page and we won't show Positions Page itself to logged out users